x-pack/legacy/plugins/siem/public/components/link_to/redirect_to_network.tsx

```diff
@@ -10,17 +10,17 @@ import { RouteComponentProps } from 'react-router-dom';
 import { RedirectWrapper } from './redirect_wrapper';
 
 export type NetworkComponentProps = RouteComponentProps<{
-  ip: string;
+  detailName: string;
```

Comment:
I think there are pros and cons to generalise the `detailName`. It looks quite clean but it seems that if we are going to have more than one kind of `detailName` under each page, we have to set up some logic to check it again.

Reply:
You might be right for the future, at this point, we do not have this problem

Reply:
I think simplicity at the beginning is a better option than over-engineering at the beginning